test/rewards/AutoCompoundingRewards.ts

```diff
@@ -634,7 +639,50 @@ describe('AutoCompoundingRewards', () => {
                     });
                 });
 
-                describe('processing rewards', () => {
+                describe('the number of programs to auto-process the rewards for', () => {
+                    it('should initially be larger than zero', async () => {
+                        expect(await autoCompoundingRewards.autoProcessRewardsCount()).to.be.gt(0);
```

Comment:
This should be part of the `construction` test group

Reply:
done